planet/cli/data.py

```diff
@@ -135,6 +135,22 @@ async def search_create(ctx, name, item_types, filter, daily_email, pretty):
         echo_json(items, pretty)
 
 
+@data.command()
+@click.pass_context
+@translate_exceptions
+@coro
+@pretty
+@click.argument('search_id')
+async def search_get(ctx, search_id, pretty):
+    """ This function obtsains an existing saved search, using the search_id.
+    This function outputs a full JSON description of the identified saved
+    search. The output can also be optionally pretty-printed using "--pretty".
+    """
+    async with data_client(ctx) as cl:
+        items = await cl.get_search(search_id=search_id)
```

Comment:
search_id is a required, positional argument. there is no need to specify it as a keyword arg

```suggestion
        items = await cl.get_search(search_id)
```